### ISSUE TITLE
Allow überbar to have custom actions

### ISSUE THREAD

**Issue by rthrash** [modxcms/revolution#11116]
As a content author it would be awesome to perform custom actions  from the uberbar so I can get work done faster and interact in any way I choose from the CMS.

Examples of desired functionality:
- Create new article under a particular parent or Collections container
- search ONLY for a doc under a particular parent…
- trigger opening a custom 3PC installed in our site

**Comment by Ibochkarev** [comment]
### Triage (current `3.x`)

Still an open **proposal**. Überbar remains a fixed object search, not a command palette for custom actions.

#### What exists today

- UI: `modx.searchbar.js` → `Search/Search`
- Backend: `Processors/Search/Search.php` only searches resources / chunks / templates / TVs / snippets / plugins / users (permission-gated). No hook, no action registry, no “create under parent X” shortcuts.
- The dropdown template still special-cases `type: 'actions'` (icon + `search_resulttype_actions` lexicon), but the processor **never** emits that type — leftover from the old “search manager actions” idea after `modAction` went away (see @rtripault’s 2014 note on this issue).

Workarounds called out in-thread (dashboard widget / button) still apply; they are outside überbar.

#### Overlap with newer work

| Item | Relation |
| --- | --- |
| [#16912](https://github.com/modxcms/revolution/issues/16912) / open PR [#16915](https://github.com/modxcms/revolution/pull/16915) (`OnManagerSearch`) | Lets plugins **append search results** (custom object types / CMP entry points via `_action`). Closest path for “open our 3PC from überbar.” |
| [#11117](https://github.com/modxcms/revolution/issues/11117) | Query DSL (`in:100`, type excludes) — covers “search only under a parent,” not custom create actions. |

`OnManagerSearch` does **not** by itself give first-class “Create article under Collections container 42” commands unless an Extra (or core) registers those as synthetic results matching the query.

#### Mapping the original examples

1. **Create under a particular parent / Collections** — still missing as core UX (command results or dedicated create shortcuts).
2. **Search only under a parent** — belongs with #11117 / scoped resource query, not custom actions.
3. **Open a custom 3PC** — doable once #16915 (or equivalent) lands: plugin returns `{ name, type, _action: 'namespace:…' }` rows.

#### Suggested disposition

- Treat **#16912 / #16915** as the extension mechanism for Extras-driven results and CMP deep-links.
- Keep this issue open for the **command / create-action** half (or narrow the title to that), or close as superseded once maintainers decide commands are out of scope for überbar.
- Drop or wire up the unused `actions` result type so UI/lexicon match reality.

`urgent` on a 2014 RFC looks stale relative to the concrete #16915 PR.
